### PR TITLE
Support authorization for helm fetch

### PR DESCRIPTION
**Pull request by dblbirdie**
This PR enables support for chart repositories that are not public. If they need authentication, credentials can now be provided via `--username` and `--password`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,3 +60,5 @@ wrote ./jenkins/templates/jenkins-agent-svc.yaml
 wrote ./jenkins/templates/jenkins-master-svc.yaml
 wrote ./jenkins/templates/jenkins-master-deployment.yaml
 ```
+
+If the chart repository needs authentication, provide credentials via `--username` and `--password`.
```
Comment (torstenwalter) [inline README.md:64]: As we are using `viper.AutomaticEnv()` environment variables should also work. Could you verify this manuall and also document it here in the README?

File: pkg/helmt/helmt.go [hunk 2]
```diff
@@ -129,8 +129,18 @@ func template(name string, chart string, values []string, namespace string, skip
...
-	return execute("helm", execOpts{}, "fetch", "--repo", repository, "--version", version, chart)
+func fetch(repository, chart, version string, username, password string) error {
+	args := []string{"fetch"}
+	args = append(args, "--repo", repository)
+	args = append(args, "--version", version)
+	if username != "" {
+		args = append(args, "--username", username)
+	}
+	if password != "" {
+		args = append(args, "--password", password)
```
Comment (viktor-yakubiv) [inline pkg/helmt/helmt.go:140]: Should there be any tying between username and password? I.e., when you pass one without another, should the tool give you a nice feedback?

File: README.md (hunk 1 above)
Comment (dblbirdie) [reply README.md:64]: Currently, we do not bind flags and only the command line parameters will work.

File: pkg/helmt/helmt.go (hunk 2 above)
Comment (dblbirdie) [reply pkg/helmt/helmt.go:140]: I checked how `helm` reacts if you provide only one of the two parameters. There is no special feedback. So I did not implement additional logic for `helmt` either.

**Review by a-hat — APPROVED** [review]
LGTM

**Pull request merged**